docs/developers/meta/rust-nightly.md

```diff
@@ -1,189 +1,44 @@
-        </p>
-        <p>
-          Alternately, we could wait for an overlapping marker trait impls.
-        </p>
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>try_trait_v2</code>
-      </td>
-      <td>
-        <p>
-          Used in the deprecated <code>SCResult</code> type.
-        </p>
-      </td>
-      <td>
-        <p>
-          Remove <code>SCResult</code>.
-        </p>
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>maybe_uninit_uninit_array</code> and <code>maybe_uninit_array_assume_init</code>
-      </td>
-      <td>
-        <p>
-          Used in <code>ManagedVec</code> <code>to_array_of_refs</code>.
-        </p>
-      </td>
-      <td>
-        <p>
-          There are less elegant ways to write that method, without using these features.
-        </p>
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>generic_const_exprs</code>
-      </td>
-      <td>
-        <p>
-          Used everywhere in working with the <code>ManagedVec</code> payload, without using the heap. They allow us to allocate variable sized byte arrays on the stack.
-        </p>
-      </td>
-      <td>
-        <p>
-          Using universal fixed-sized arrays on heap for handling payloads. Might hurt performance, and might also occasionally run into overflows.
-        </p>
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>slice_partition_dedup</code>
-      </td>
-      <td>
-        <p>
-          Used in <code>ManagedVec</code> <code>dedup</code>.
-        </p>
-      </td>
-      <td>
-        <p>
-          We could probably replace that one with an in-house de-duplication function.
-        </p>
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>is_sorted</code>
-      </td>
-      <td>
-        <p>
-          Used in <code>ManagedVec</code> <code>is_sorted</code>, <code>is_sorted_by</code>, and <code>is_sorted_by_key</code>. These methods are currently mostly used in tests.
-        </p>
-      </td>
-      <td>
-        <p>
-          We could probably replace them with in-house equivalent functions.
-        </p>
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>panic_info_message</code>
-      </td>
-      <td>
-        <p>
-          Used in the wasm adapter to retrieve panic messages, for debugging purposes.
-        </p>
-      </td>
-      <td>
-        <p>
-          Figuring out a way to use the format function provided by the stable version of the library, without performing memory allocation.
-        </p>
-      </td>
-    </tr>
-  </table>
-);
+## Recommended compiler versions
 
-[comment]: # (mx-context-auto)
+For everything after v0.50.0 we recommend running the latest stable version of Rust. Older versions have had compatiblity issues with certain framework dependencies, on certain versions of the compiler.
 
-## Deal breakers
+Nightly Rust is still allowed, but not recommended. We will still be running 
```

Comment:
This looks like an unfinished sentence. I get the feeling something is missing after "We will still be running".